app/models/ingredient.rb

```diff
@@ -1,13 +1,12 @@
 class Ingredient < ApplicationRecord
 
-  VALID_CATEGORIES = ['veg', 'non_veg']
-  ERROR_MESSAGE = "not a valid category"
+  VALID_CATEGORIES = { veg: 'veg', non_veg: 'non_veg' }
```

Comment:
Let the value here be 'Veg' and 'Non Veg'. Then we don't have to change the value later.